sources/packages/backend/workflow/test/2023-2024/parttime-assessment/costs/parttime-assessment-2023-2024-additional-transportation-costs.e2e-spec.ts

```diff
@@ -0,0 +1,339 @@
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataTotalTransportationAllowance,
+      ).toBe(469.2);
+    },
+  );
+
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm less than 280 " +
+      "and studentDataAdditionalTransportPlacement is false and offeringDelivered is onsite",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 100;
+      assessmentConsolidatedData.offeringWeeks = 30;
+      assessmentConsolidatedData.studentDataAdditionalTransportWeeks = 20;
+      assessmentConsolidatedData.studentDataAdditionalTransportCost = 80;
+      assessmentConsolidatedData.studentDataAdditionalTransportPlacement =
+        false;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Onsite;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // calculatedDataAdditionalTransportationMax = min(94, 80) * min(30, 20) * 0.34 = 544
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataAdditionalTransportationMax,
+      ).toBe(544);
+      // calculatedDataTotalTransportationAllowance = (30 - 20) * 13 + 544 = 674
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataTotalTransportationAllowance,
+      ).toBe(674);
+    },
+  );
+
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm less than 280 " +
+      "and studentDataAdditionalTransportPlacement is false and offeringDelivered is online",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 100;
+      assessmentConsolidatedData.offeringWeeks = 30;
+      assessmentConsolidatedData.studentDataAdditionalTransportWeeks = 20;
+      assessmentConsolidatedData.studentDataAdditionalTransportCost = 80;
+      assessmentConsolidatedData.studentDataAdditionalTransportPlacement =
+        false;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Online;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // calculatedDataAdditionalTransportationMax = min(94, 80) * min(30, 20) * 0.34 = 544
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataAdditionalTransportationMax,
+      ).toBe(544);
+      // calculatedDataTotalTransportationAllowance = min(544, 80 * min(30, 20)) = 544
```

Comment:
https://github.com/bcgov/SIMS/pull/2824/files#r1491807589

Reply:
The comment is now updated to use `calculatedDataAdditionalTransportationMax`. Please have a look.